### PR TITLE
Adapter upgrades

### PR DESCRIPTION
**Pull request by the-greenman**
This PR contains all the recent updates for adapters, and the sponsor info.

### CODE REVIEW

File: cypress.json [hunk 1]
```diff
@@ -17,5 +17,6 @@
   "watchForFileChanges":false,
   "projectId": "ebzwp1",
   "pageLoadTimeout": 200000,
-  "defaultCommandTimeout": 30000
+  "defaultCommandTimeout": 30000,
+  "experimentalStudio" : true
```
Comment (lukestanley) [inline cypress.json:21]: Have you noticed if the speed on CI is impacted by this? @the-greenman

File: cypress/integration/adapters.ts [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+      fixture: 'bloomen.json'
+    }
+    ).as('bloomen.json');
+
+  });
+
+  it('Export a packaged adapter with attachments', () => {
+    // first go to settings and allow core actions in flows
+        cy.visit('/core/settings');
+        cy.get('#formly_16_boolean_exposeCoreActions_5 .mat-checkbox-inner-container').click();
```
Comment (lukestanley) [inline cypress/integration/adapters.ts:24]: I wonder how well this selector will hold.

**Review by lukestanley — APPROVED** [review]

File: cypress.json (hunk 1 above)
Comment (the-greenman) [reply cypress.json:21]: I didn't see any change. It almost looked faster.

File: cypress/integration/adapters.ts (hunk 2 above)
Comment (the-greenman) [reply cypress/integration/adapters.ts:24]: Took a while to find a selector that eventually worked. I failed to get it to work with a "contains" statement, as there are a number of nested elements, that seemed to get in the way. I hope that the ID of the field will be fairly static.

**Pull request merged**